mtags/src/main/scala-2/scala/meta/internal/pc/completions/OverrideCompletions.scala

```diff
@@ -213,6 +213,16 @@ trait OverrideCompletions { this: MetalsGlobal =>
         if (sym.isLazy) "lazy "
         else ""
 
+      val _modifs = sym.flagString
+        .replace("package ", "")
```

Comment:
Do we need to do this? We are not doing overrides for any of those symbol types.

Reply:
```
Not sure if this is a bug or intended behaviour but flagString generates modifiers like protected[package x.y.z] and they need to be stripped manually.
```

Reply:
What about `object`, `trait` or `class` ?

Reply:
handled already

Reply:
I meant, does those keywords can appear here even? You mentioned `protected[package x.y.z] `, but why would trait appear here?

Reply:
```
scala> trait A {
     |   trait AB {
     |     private[A] val x = 3
     |   }
     | }
trait A
```

Reply:
We could do:
```
      val _modifs =
        sym.flagString.replace(
          sym.privateWithin.toString(),
          sym.privateWithin.name.toString()
        )
```